src/app/lista-nominal/page.tsx

```
@@ -0,0 +1,156 @@
+            token: params.token,
+            listName: params.listName,
+            cardType: "external",
+            ine: params.ine,
+            baseUrl: baseURL() ,
+        });
+
+        externalCardsProps = getCardsProps(cardsDetails, data);
+    } catch (error) {
+        captureException(error);
+        console.log(error)
+        return <p>Erro ao buscar dados cards</p>;
```

Biome (1.9.4)

[error] Don't use console.log

console.log is usually a tool for debugging and you don't want to have that in production.
If it is not for debugging purpose then using console.info might be more appropriate.
Unsafe fix: Remove console.log


(lint/suspicious/noConsoleLog)